src/CMakeLists.txt

```diff
@@ -59,28 +61,16 @@ if (NOT CONFIG_OSDP_STATIC_PD)
 	)
 endif()
 
-list(APPEND LIB_OSDP_INCLUDE_DIRS
-	${PROJECT_SOURCE_DIR}/utils/include
-	${PROJECT_SOURCE_DIR}/include
-)
-
 list(APPEND LIB_OSDP_PRIVATE_INCLUDE_DIRS
 	${CMAKE_CURRENT_SOURCE_DIR}
 	${CMAKE_CURRENT_BINARY_DIR}
 )
 
-list(APPEND LIB_OSDP_LIBRARIES
-	utils
-)
-
 if (OpenSSL_FOUND)
-	list(APPEND LIB_OSDP_INCLUDE_DIRS ${OPENSSL_INCLUDE_DIR})
-	list(APPEND LIB_OSDP_LIBRARIES ${OPENSSL_CRYPTO_LIBRARY})
+	list(APPEND LIB_OSDP_LIBRARIES ${OPENSSL_CRYPTO_LIBRARY}) # Needed for python library building
```

Comment:
Both `OpenSSL_FOUND` and  `MbedTLS_FOUND ` branches must set,

```
list(APPEND LIB_OSDP_PRIVATE_INCLUDE_DIRS ${OPENSSL_CRYPTO_INCLUDE_DIRS})
```
and
```
list(APPEND LIB_OSDP_PRIVATE_INCLUDE_DIRS ${MBEDTLS_INCLUDE_DIRS})
```
respectively.

Reply:
Also, looks like LIB_OSDP_INCLUDE_DIRS below (L84) is no longer set/used anymore. If it is needed it has to be populated.

Reply:
Also, comment "Needed for python library building" is not necessary. this interface is not just for python library but a any consumer within this project.

Reply:
The problem is that this fails to link cleanly when libosdp is built under vcpkg.  Because all of the dependencies are built by vcpkg and not installed in the system, the only correct way to link with OpenSSL is to tell CMake to link with `OpenSSL::Crypto`.  

The comment is there because the only place where LIB_OSDP_LIBRARIES is used is with the python building(which as far as I can tell only uses OpenSSL).

Reply:
Okay, for now, let us merge this PR by apply the fixup that I posted so things don't break. I'll add libosdp to vckpg upstream soon(ish).

Reply:
> what problem are you seeing where things break?

Can you take a look at the [CI failure](https://github.com/goToMain/libosdp/pull/108/checks)? I can reproduce that locally too. Your system local changes makes things work for you.

> The problem that I have is that I can't apply your fix, because then my builds will fail

I meant for the time being let us merge this branch with that fixup (fixing CI builds). Then I will take some time to investigate vcpkg integration issue that you are facing. But if you can fix the CI, I'm okay with merging it.